### DISCUSSION TITLE
Failed to connect via WMI & RPC Server is not available

### DISCUSSION THREAD

**Discussion by codetangler** [belowaverage-org/SuperGrate#124] [Q&A] [answered]
I'm getting the following errors when trying to list profiles from a number of the computers in our AD domain:
"Failed to connect to [hostname] via WMI. Make sure the remote PC is online."
"The RPC server is unavailable. (Exception from HRESULT:  0x800706BA)"

Every computer that I am experiencing this issue with has no ohter symptoms. Active Directory and other tools relying on it work flawlessly. I have made sure the DCOM service is running and the "Remote Assistance DCOM-In" Firewall rule is enabled.
Do you have any pointers for me?

**Comment by krisdb2009** [comment]
Try disabling the windows firewall one one remote PC, and if that works, you know something is being blocked and work backwards from there.

**Reply by codetangler** [reply]
Thanks, after disabling the windows firewall it works as expected. Do you have any idea which firewall rule could be impeding this? There are only default winodws firewall rules on that machine.

**Reply by codetangler** [reply, marked as answer]
I figured it out. The "Windows Management Instrumentation DCOM In" firewall rule was disabled. I guess my earlier searches about the RPC error led me to the Remote Assistance rule, and that was not the issue. Problem solved now though!
